aiida/manage/configuration/schema/config-v5.schema.json

```diff
@@ -0,0 +1,357 @@
+                    "key": "logging_paramiko_log_level",
+                    "type": "string",
+                    "enum": [
+                        "CRITICAL",
+                        "ERROR",
+                        "WARNING",
+                        "REPORT",
+                        "INFO",
+                        "DEBUG"
+                    ],
+                    "default": "WARNING",
+                    "description": "Minimum level to log to daemon log and the `DbLog` table for the `paramiko` logger"
+                },
+                "logging.alembic_loglevel": {
+                    "type": "string",
+                    "enum": [
+                        "CRITICAL",
+                        "ERROR",
+                        "WARNING",
+                        "REPORT",
+                        "INFO",
+                        "DEBUG"
+                    ],
+                    "default": "WARNING",
+                    "description": "Minimum level to log to daemon log and the `DbLog` table for the `alembic` logger"
+                },
+                "logging.sqlalchemy_loglevel": {
+                    "type": "string",
+                    "enum": [
+                        "CRITICAL",
+                        "ERROR",
+                        "WARNING",
+                        "REPORT",
+                        "INFO",
+                        "DEBUG"
+                    ],
+                    "default": "WARNING",
+                    "description": "Minimum level to log to daemon log and the `DbLog` table for the `sqlalchemy` logger"
+                },
+                "logging.circus_loglevel": {
+                    "type": "string",
+                    "enum": [
+                        "CRITICAL",
+                        "ERROR",
+                        "WARNING",
+                        "REPORT",
+                        "INFO",
+                        "DEBUG"
+                    ],
+                    "default": "INFO",
+                    "description": "Minimum level to log to daemon log and the `DbLog` table for the `circus` logger"
+                },
+                "warnings.showdeprecations": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Boolean whether to print AiiDA deprecation warnings"
+                },
+                "transport.task_retry_initial_interval": {
+                    "type": "integer",
+                    "default": 20,
+                    "minimum": 1,
+                    "description": "Initial time interval for the exponential backoff mechanism."
+                },
+                "transport.task_maximum_attempts": {
+                    "type": "integer",
+                    "default": 5,
+                    "minimum": 1,
+                    "description": "Maximum number of transport task attempts before a Process is Paused."
+                },
+                "caching.default": {
```

Comment:
this we discussed in a separate comment

Reply:
TODO: see https://github.com/aiidateam/aiida-core/pull/4712#issuecomment-777418357